exporter/stackdriverexporter/observability.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	pointCount = stats.Int64("otelcol/googlecloudmonitoring/point_count", "Count of metric points written to Cloud Monitoring.", "1")
+	pointCount = stats.Int64("googlecloudmonitoring/point_count", "Count of metric points written to Cloud Monitoring.", "1")
```

Comment:
It may have been useful to change prefix to `exporter/googlecloudmonitoring/...` as the standard collector metrics are prefixed as `otelcol_exporter_...`, `otelcol_receiver_...`, i.e. they include the "component type"